mirobo/wifispeaker.py

```diff
@@ -0,0 +1,79 @@
+import warnings
+import logging
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
The logger isn't used?

Reply:
It is not currently, but I think it's ok to leave it like that for now. This will probably need some revisions to be remotely useful, and for that there'll be debug outputs I hope.